libs/ledger-live-common/src/account/serialization.ts

```diff
@@ -797,15 +797,23 @@ export function fromAccountRaw(rawAccount: AccountRaw): Account {
         (rawAccount as BitcoinAccountRaw).bitcoinResources
       );
       break;
-    case "algorand":
-      (res as AlgorandAccount).algorandResources = fromAlgorandResourcesRaw(
-        (rawAccount as AlgorandAccountRaw).algorandResources
-      );
+    case "algorand": {
+      const algoResourcesRaw = (rawAccount as AlgorandAccountRaw)
+        .algorandResources;
+      if (algoResourcesRaw)
```

Comment:
Do we really need to do this ? If so, would it make sense to go back to the previous wait with multiple ifs ? Or maybe apply it to every families

Reply:
Either way we need to chose one, but yeah we do need to do this because this implementation doesn't seems to work well with tests at all, most of the time we wont have any familyResources